modules/stanford_profile_styles/lib/scss/stanford_profile_styles.scss

```diff
@@ -80,10 +80,10 @@ a {
     }
   }
 
-  svg {
 
-    // Style the SVG for mailto.
-    &.mailto {
+  // Style the SVG for mailto.
+  &.mailto {
```

Comment:
This seems fine to me, but the styles aren't in the wysiwyg. Do we want to work on that or we can make that another ticket.

Reply:
Thanks for your comment, @pookmish. 
 If I create a button through the WYSIWYG and use an email link, I see the icon show up with the color change. Please help me understand when you say the styles aren't in the WYSIWYG?

Reply:
![image](https://user-images.githubusercontent.com/7185045/97911262-d4f50e00-1cff-11eb-9ae4-c664c3411b7b.png)

Reply:
@pookmish,

Ahhh, that makes sense. Where do we go to make this magic happen?

Reply:
https://github.com/SU-SWS/stanford_profile_helper/blob/8.x-1.x/modules/stanford_profile_styles/lib/scss/admin/ckeditor.scss
The issue is that the classes and stuff generated by the external links module won't be in the wysiwyg. You might have to fake it.

Reply:
How about I start a new PR for the WYSIWYG styles?

Reply:
that sounds good. Please create a ticket as well.

Reply:
https://stanfordits.atlassian.net/browse/D8CORE-2950